docs/cli/restapi/restapi.md

```diff
@@ -3,35 +3,139 @@ title: Overview
 description: Use Amplify CLI's simple guided workflow to add REST APIs to cloud-based web and mobile apps.
 ---
 
-## Set up a REST API with NoSQL database
+The Amplify CLI provides a guided workflow to easily add, develop, test and manage REST APIs to access your AWS resources from your web and mobile applications. 
 
-During the CLI setup, you'll be guided through to create a new Lambda function with a predefined [serverless-express](https://github.com/awslabs/aws-serverless-express) template with routing enabled for your REST API paths with support for CRUD operations to DynamoDB tables (which you can create by following the CLI prompts or use the tables which you've already configured using the `amplify add storage` command).
+A REST API or HTTP endpoint will be composed by one or more paths. Eg: `/items`. Each path will use a Lambda function to handle HTTP requests and responses. Amplify CLI creates a single resource in Amazon API Gateway so you can handle all routes, HTTP Methods and paths, with a single Lambda function via a Lambda Proxy integration. HTTP proxy integrations forward all requests and responses directly through to your HTTP endpoint.
+
+Amplify CLI let's you choose either an existing Lambda function or create a new one. To kickstart your implementation, you can choose between the following templates: 
+- Serverless ExpressJS function
+- CRUD function for DynamoDB
+
+> Lambda templates use [serverless-express](https://github.com/awslabs/aws-serverless-express) and provide the building blocks to start your REST API development.
+
+Amplify CLI supports the following Lambda runtimes:
```

Comment:
I'd compress this into a sentence and point to the functions docs for the full list. This way when we add a new runtime, this doc will still be up-to-date.

Reply:
See changes.